packages/eslint-config/package.json

```diff
@@ -7,7 +7,7 @@
     "@typescript-eslint/eslint-plugin": "^5.52.0",
     "@typescript-eslint/parser": "^5.52.0",
     "eslint-config-prettier": "^8.6.0",
-    "eslint-config-turbo": "^0.0.7",
```

Comment:
@cesarenaldi @krzysu is `eslint-config-turbo used at all in our monorepo setup?

Reply:
good point, I don't see anything like `"extends": ["turbo"]` in the codebase, also not sure what it is even doing.

Reply:
I will remove it and see what will happen :)